sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -56,15 +56,16 @@ az cognitiveservices account create \
     --yes
 ```
 ### Authenticate the client
-In order to interact with the Text Analytics service, you'll need to create an instance of the [TextAnalyticsClient](#create-a-client) class. You would need an **endpoint** and **subscription key** to instantiate a client object.
+In order to interact with the Text Analytics service, you'll need to create an instance of the 
+`TextAnalyticsClient` class. You would need an **endpoint** and **API key** to instantiate a client object.
 
 #### Get credentials
 ##### Types of credentials
-The `subscriptionKey` parameter may be provided as the subscription key to your resource or as a token from Azure Active Directory.
+The authentication credential may be provided as the API key to your resource or as a token from Azure Active Directory.
 See the full details regarding [authentication](https://docs.microsoft.com/azure/cognitive-services/authentication) of 
 cognitive services.
 
-1. To use a [subscription key](https://docs.microsoft.com/azure/cognitive-services/cognitive-services-apis-create-account?tabs=multiservice%2Cwindows#get-the-keys-for-your-resource), 
+1. To use a [API key](https://docs.microsoft.com/azure/cognitive-services/cognitive-services-apis-create-account?tabs=multiservice%2Cwindows#get-the-keys-for-your-resource), 
    provide the key as a string. This can be found in the Azure Portal under the "Quickstart" 
```

Comment:
not related to your change, but it will be good to add the link to the Azure Portal